gemfire-shared/src/main/java/com/gemstone/gemfire/internal/shared/Version.java

```diff
@@ -227,11 +229,17 @@
   public static final Version GFXD_20 = new Version("GFXD", "2.0",
       (byte)2, (byte)0, (byte)0, (byte)0, GFXD_20_ORDINAL, GFE_75);
 
+  private static final byte GFE_80_ORDINAL = GFXD_14_ORDINAL;
+  public static final Version GFE_80 = new Version("GFE", "8.0", (byte)8,
+      (byte)0, (byte)0, (byte)0, GFE_80_ORDINAL);
+
   private static final byte GFXD_155_ORDINAL = 32;
 
   /** GemFire version is at 7.1 for compatibility with external clusters. */
   public static final Version GFXD_155 = new Version("GFXD", "1.5.5",
-      (byte)1, (byte)5, (byte)5, (byte)0, GFXD_155_ORDINAL, GFE_71);
+      (byte)1, (byte)5, (byte)5, (byte)0, GFXD_155_ORDINAL, GFE_80);
```

Comment:
Ok got it. So the problem really is that PutAllOp on client side in Snappy-store is 7.5 one which is compatible with 8.0.